src/sentry/stacktraces/processing.py

```diff
@@ -559,6 +564,12 @@ def process_stacktraces(data, make_processors=None, set_raw_stacktrace=True):
                     processing_task, stacktrace_info, processable_frames
                 )
                 if new_frames is not None:
+                    for processor in processing_task.iter_processors():
+                        # Make any necessary list-minute adjustments
+                        new_frames, new_raw_frames, errors = processor.postprocess_step(
+                            new_frames, new_raw_frames, errors
+                        )
```

Comment:
Open question for reviewers: Here in `process_stacktraces`, we can't seem to decide if there will be one processor or multiple processors. (Just above here, we iterate through `processing_task.iter_processors()` if the event is an exception, but then we only call `process_single_stacktrace` once, relying on it calling each frame's single processor.) 

To be on the safe side, I used `iter_processors()` - didn't want to miss a processor - but at the same time, that makes no sense, because the processed stacktrace has only been influenced by a single processor, so shouldn't it only get post-processed by that processor and no others?

The alternative is to pull the processor off of one of the members of `processable_frames`, which then forces the assumption that all frames are using the same processor (which, of course they are, why wouldn't they be, but then why does each frame get its own pointer?).

Would love some clarity and/or opinions here. Am I right to be scratching my head at this? And which is the better way to get ahold of the processor?

Reply:
UPDATE: I may end up answering some of my own questions as I get tests passing. I'll come back to this.